demos/src/main.mustache

```diff
@@ -1,426 +1,135 @@
-											</li>
-
-											<li class="o-header__meganav-item o-header__meganav-section--mobile">
-												<a class="o-header__meganav-link" href=/special-reports>Special Reports</a>
-											</li>
-
-											<li class="o-header__meganav-item o-header__meganav-section--mobile">
-												<a class="o-header__meganav-link" href=http://portfolio.ft.com/dashboard/?ft_site&#x3D;falcon>Portfolio</a>
-											</li>
-
-											<li class="o-header__meganav-item o-header__meganav-section--mobile">
-												<a class="o-header__meganav-link" href=/stream/sectionsId/MTQw-U2VjdGlvbnM&#x3D;>Personal Finance</a>
-											</li>
-
-											<li class="o-header__meganav-item o-header__meganav-section--mobile">
-												<a class="o-header__meganav-link" href=https://ftepaper.ft.com>Today&#x27;s Editions / ePaper</a>
-											</li>
-											
-											<!-- TODO: Move from meganav-section to meganav-item and meganav-section-head to meganav-link and apply that to these tools -->
-											<li class="o-header__meganav-item o-header__meganav-section--mobile">
-												<a class="o-header__meganav-link" href="/myft" aria-label="my F T"><i>my</i>FT</a>
-											</li>
-
-											<li class="o-header__meganav-item o-header__meganav-section--mobile">
-												<a class="o-header__meganav-link" href="/login">Sign In</a>
-											</li>
-										  
-											<li class="o-header__meganav-item o-header__meganav-section--mobile">
-												<a class="o-header__meganav-link" href="/logout">Sign Out</a>
-											</li>
-										  
-											<li class="o-header__meganav-item o-header__meganav-section--mobile">
-												<a class="o-header__meganav-link" href="http://registration.ft.com/registration/selfcare/">My Account</a>
-											</li>
-
-											<li class="o-header__meganav-item o-header__meganav-section--mobile">
-												<a class="o-header__meganav-link" href="/products?segD=400863&amp;segmentID=190b4443-dc03-bd53-e79b-b4b6fbd04e64">Subscribe</a>
-											</li>
-										</ol>
-									</div>
+				<div class="o-header__masthead-mobile">
+					<a href="#" title="Go to Financial Times homepage"><span>Financial Times</span></a>
+				</div>	
+
+				<nav class="o-header__nav-container" role="navigation" aria-label="Primary navigation">
+					<ol class="o-header__nav">
+						<li class="o-header__nav-item o-header__meganav-container">
+							<!-- Links to a core experience nav outside of the header -->
+							<a class="o-header__nav-link o-header__nav-link--toggle o--if-no-js" href="#">
+								<span class="o-header__meganav-toggle o-header__meganav-toggle--open">Menu</span>
+							</a>
+
+							<div class="o--if-js">
+								<div class="o-header__nav-container">
+									<button class="o-header__nav-link o-header__nav-link--toggle" data-o-header-togglable data-o-header-togglable-nav>
+										<span class="o-header__meganav-toggle o-header__meganav-toggle--open">Menu</span>
```

Comment:
I think it currently works fine with voiceover like it is, so I'm not 100% sure it's necessary, and it might makes things a little dirtier. Could you raise an issue about this so we can discuss it outside of the pr please?